src/Metadata/class-post-builder.php

```diff
@@ -570,6 +570,17 @@ private function get_categories( int $post_id, string $delimiter = '/' ): array
 	private function get_custom_taxonomy_values( WP_Post $post_obj ): array {
 		// Filter out default WordPress taxonomies.
 		$all_taxonomies = array_diff( get_taxonomies(), array( 'post_tag', 'nav_menu', 'author', 'link_category', 'post_format' ) );
+
+		/**
+		 * Filter the taxonomies.
+		 *
+		 * @since
```

Comment:
```suggestion
		 * @since 3.11.0
```
Let's release this ASAP!